types/framework/entities/item.d.ts

```diff
@@ -131,28 +141,28 @@ declare class Item<D extends Item.Data = Item.Data<any>> extends Entity<D> {
   /* -------------------------------------------- */
 
   /** @override */
-  update(data: Optional<D>, options?: Entity.UpdateOptions): Promise<this>;
+  update(data: DeepPartial<D>, options?: Entity.UpdateOptions): Promise<this>;
 
   /** @override */
   delete(options?: Entity.DeleteOptions): Promise<Item>;
 
   /**
    * A convenience constructor method to create an Item instance which is owned by an Actor
-   * @param itemData -
-   * @param actor -
    */
-  static createOwned(itemData: Item.Data, actor: Actor): Item;
+  static createOwned(itemData: DeepPartial<Item.Data>, actor: Actor): Item;
 }
 
 declare namespace Item {
   /**
-   * Typing for the data.data field
+   * @typeParam D - Type for Item.data.data
    */
-  type DataData<T> = T extends Data<infer D> ? D : never;
-
   interface Data<D = any> extends Entity.Data {
     data: D;
+    effects: ActiveEffect[];
```

Comment:
This is incorrect, it's an array of the data of an active effect. However, this is adjusted as part of my PR: #216